### PR TITLE
Rework the interface for a simple Phenix.rise! and Phenix.burn!

### PR DESCRIPTION
**Pull request by pschambacher**
:octopus::computer:

/cc @grosser @bquorning 

### Description
Simpler to use. No need to include the module in your spec, just use class methods.
Also use Grosser's workaround `/gemfiles` by replacing `Bundler.root` with `Dir.pwd`.

### Risks
* None

### CODE REVIEW

**Comment by pschambacher** [comment]
I updated https://github.com/zendesk/active_record_host_pool/pull/18 accordingly.

File: lib/phenix.rb [hunk 1]
```diff
@@ -3,23 +3,36 @@
...
-    attr_accessor :database_config_path, :schema_config_path
+    attr_accessor :database_config_path, :schema_path, :skip_database
     attr_accessor :current_configuration
 
     def configure
-      self.database_config_path = File.join(Bundler.root, 'test', 'database.yml')
-      self.schema_config_path   = File.join(Bundler.root, 'test', 'schema.rb')
+      self.database_config_path = File.join(Dir.pwd, 'test', 'database.yml')
+      self.schema_path          = File.join(Dir.pwd, 'test', 'schema.rb')
+      self.skip_database        = nil
```
Comment (bquorning) [inline lib/phenix.rb]: You could default to `->{ false }` and remove the `nil` check in line 61.

Comment (pschambacher) [inline lib/phenix.rb]: Yeah... could do

File: lib/phenix.rb [hunk 2]
```diff
@@ -3,23 +3,36 @@
...
 
       yield(self) if block_given?
     end
   end
 
+  def rise!(with_schema: true, config_path: Phenix.database_config_path)
+    load_database_config(config_path)
+    drop_databases
+    with_schema ? create_and_populate_databases : create_databases
```
Comment (bquorning) [inline lib/phenix.rb]: Could this be a bit simpler?
```rb
create_databases
populate_databases if with_schema
```

Comment (pschambacher) [inline lib/phenix.rb]: I could pass `with_schema` to `create_database` and have `populate_database if with_schema`

Comment (bquorning) [inline lib/phenix.rb]: does `create_database` need to know about `with_schema`?

**Comment by bquorning** [comment]
:+1:

File: README.md [hunk 3]
```diff
@@ -24,22 +24,20 @@ Or install it yourself as:
...
   end
 
   # in your test file
-  include Phenix
-
   before do
-    load_database_config
-    create_databases # use this one if you just want the databases
-    create_and_populate_databases # use this one if you also want to use the schema
+    Phenix.rise! # pass with_schema: true if you want the schema populated as well
```
Comment (grosser) [inline README.md]: why would you want a db without a schema ... maybe flip this around ?

Comment (grosser) [inline README.md]: default is true ... so the docs are wrond

Comment (pschambacher) [inline README.md]: Yeah doc needs fixing

**Pull request merged**